src/lib/Scenes/Home/Components/Sales/index.tsx

```diff
@@ -25,69 +42,93 @@ const SectionListStyles = StyleSheet.create({
 
 class Sales extends React.Component<Props> {
   get data() {
-    const { sales } = this.props
+    const { sales, viewer: { sale_artworks } } = this.props
     const liveAuctions = sales.filter(a => !!a.live_start_at)
     const timedAuctions = sales.filter(a => !a.live_start_at)
 
     return {
       liveAuctions,
       timedAuctions,
+      sale_artworks,
     }
   }
 
   render() {
+    const { relay: { hasMore, isLoading, loadMore } } = this.props
+
     const sections = [
       {
         data: [{ data: this.data.liveAuctions }],
         title: "Current Live Auctions",
-        renderItem: itemData => <SaleList {...itemData} />,
+        renderItem: props => <SaleList {...props} />,
       },
       {
         data: [{ data: this.data.timedAuctions }],
         title: "Current Timed Auctions",
-        renderItem: itemData => <SaleList {...itemData} />,
+        renderItem: props => <SaleList {...props} />,
       },
       {
-        data: [{ data: [] }],
+        data: [{ data: this.data.sale_artworks }],
         title: "Lots by Artists You Follow",
-        renderItem: ({ title }) => <LotsByFollowedArtists title={title} />,
+        renderItem: props => <LotsByFollowedArtists saleArtworks={props.item.data} isLoading={isLoading} />,
       },
     ]
 
+    const loadMoreLots = () => hasMore() && loadMore(LOTS_BY_FOLLOWED_ARTISTS_PAGE_SIZE, x => x)
+
     return (
       <SectionList
         contentContainerStyle={SectionListStyles.contentContainer}
         stickySectionHeadersEnabled={false}
         sections={sections}
         keyExtractor={(item, index) => item.id}
+        onScroll={isCloseToBottom(loadMoreLots)}
```

Comment:
When attempting to use a `<FlatList />` + end detection in these nested containers it would incorrectly state that we've reached the bottom and constantly refetch. I spent a ton of time playing with it but couldn't figure out why. In the interest of time went with `onScroll` + new util as seen in various other places in app. (Can return later to figure out)